packages/ckeditor5-package-generator/tests/utils/copy-files.js

```diff
@@ -143,7 +143,7 @@ describe( 'lib/utils/copy-files', () => {
 		expect( stubs.logger.process.firstCall.firstArg ).to.equal( 'Copying files...' );
 	} );
 
-	it( 'creates the ".gitignore" file', () => {
+	it( 'creates the ".gitignore" file for javascript', () => {
```

Comment:
```suggestion
	it( 'creates the ".gitignore" file for JavaScript', () => {
```

Please, make sure the tool uses the `JavaScript` name everywhere instead of `javascript`.